<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PKS_Test</class>
 <widget class="QMainWindow" name="PKS_Test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PKS Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_core">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>561</width>
      <height>581</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Run Test</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_LnLStats">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Log Likelihood Scores</string>
      </property>
      <widget class="QLabel" name="label_max">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Mean:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_mean">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Variance:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_max">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_mean">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_parameters">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>411</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Parameters</string>
      </property>
      <widget class="QFrame" name="frame_percentage">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>191</width>
         <height>151</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>0</y>
          <width>201</width>
          <height>151</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_percentage">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Percentage of trees to be analysed:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_standard">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>standard (30%)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_other">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Other:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_other">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_confLevel" native="true">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>191</width>
         <height>151</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_confLevel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Significance Level:</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_90">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0.90</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_95">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0.95</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_99">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>100</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0.99</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_KSTest">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>531</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Kolmogorov-Smirnov Test</string>
      </property>
      <widget class="QLabel" name="label_DValue">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>D value:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_dValue">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_conclusion">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Conclusion:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_TreeQuant">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantity of trees analysed:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_TreeQuant">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>50</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="critical_label">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>80</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Critical Value:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="critical_line">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>80</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_conc">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>411</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>260</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Observed Distribution Parameters</string>
      </property>
      <widget class="QLabel" name="label_lambda">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Mean:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_varMut">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Variance:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_lambda">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_varMut">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Software developed at Laboratory of Evolutionary Genomics and Biocomplexity (UNIFESP)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_showGraph">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>210</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_open">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>531</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Input</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_open">
         <property name="text">
          <string>Open File:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_open">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_open">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>90</y>
        <width>111</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </widget>
    
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
